.github/workflows/android.yml

```diff
@@ -0,0 +1,22 @@
+name:  Android build CI
+
+on: pull_request 
+
+jobs:
+ build-app:
+   runs-on: ubuntu-latest
+   steps:
+     - name: Set up JDK
+       uses: actions/setup-java@v1
+       with:
+         java-version: 1.8
+     
+     - name: Clone Repo
+       uses: actions/checkout@v2
+
+      
+     - name: Build App
+       run: |
+          cd TrailGage
+          ./gradlew build --stacktrace
```

Comment:
Almost done (I think)!

In your [action](https://github.com/DIT112-V20/group-11/runs/1336596228?check_suite_focus=true), you are getting the following error:

> /home/runner/work/_temp/8768e203-8951-4670-b2ab-4e0a389898e8.sh: line 2: ./gradlew: Permission denied

This is very common and the easiest solution is to make the `gradlew` binary executable. After `cd TrailGage` you can add the following command to make `gradlew` executable before running it: `chmod +x gradlew`.

Also, can you please remove the blank lines between the steps? Unless you really like them that is. 😁